test/test_utils/sagemaker.py

```diff
@@ -267,6 +271,11 @@ def execute_sagemaker_remote_tests(image):
             context.run("pip install -r requirements.txt", warn=True)
             res = context.run(pytest_command, warn=True)
             metrics_utils.send_test_result_metrics(res.return_code)
+            if res.failed:
+                raise DLCSageMakerRemoteTestFailure(
+                    f"{pytest_command} failed with error code: {res.return_code}\n"
+                    f"Traceback:\n{res.stdout}"
```

Comment:
the "Traceback" part here doesn't appear to print out anything. Perhaps this needs to print out stderr? It looks like relevant debug information is not suppressed, so I am ok with removing this line

Reply:
Shall I fast follow with another PR?